src/main/java/dev/j3fftw/litexpansion/LiteXpansion.java

```diff
@@ -55,10 +55,10 @@ public void onEnable() {
         } catch (IllegalAccessException | NoSuchFieldException ignored) {
             getLogger().warning("Failed to register enchantment. Seems the 'acceptingNew' field changed monkaS");
         }
-        Enchantment.registerEnchantment(new GlowEnchant(Constants.GLOW_ENCHANT));
 
-        // Category
-        Items.LITEXPANSION.register();
```

Comment:
reason you removed this?

Reply:
I literally explained it in the PR body...

Reply:
lol didnt see that line